m4_sessions/1808-inning/Perspective_to_succeed

```diff
@@ -1,19 +1,63 @@
 ## Group Members:
+john livingston
+spenser leighton
+arram mandel
 
 
-
-## Name of Session: 
+## Name of Session:
+disrupting your perspective: become an (secret) agent of disruption
 
 ## Session Description 
 
 (Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
 
-## Session Outcomes 
+## Session Outcomes
+
+quote ghandi and yoda
+
+sell the benefit of disrupting your perspective, not pushing a certain perspective
+
+
 
 (What will students learn/leave with?): 
 
 ## Session Outline 
 
+
+
+count attendees as they walk into room, divide them into groups that way immediately!!!
+
+who do you like to watch lose?(10 min)
+
+introduce ourselves and something that makes us uncomfortable(3 min)
+
+have attendees talk about something that makes them uncomfortable in their groups. everyone will share.(10 min)
+
+show short clip of someone else talking about it(5 min)
+
+go into potential positives of disrupting perspective(10 min)
```

Comment:
This is great.  If you don't do slides, maybe someone is saying these benefits and the other person is writing them on the board.